etna/records/templatetags/records_tags.py

```diff
@@ -46,6 +46,31 @@ def as_label(record_field_name: str) -> str:
 
 
 @register.simple_tag
-def level_name(level_code: int) -> str:
+def level_name(level_code: int, is_tna: bool) -> str:
     """returns level as a human readable string"""
-    return LevelKeys["LEVEL_" + str(level_code)].value
+    if is_tna:
+        return LevelKeys["LEVEL_" + str(level_code)].value
+    else:
+        return NonTNALevelKeys["LEVEL_" + str(level_code)].value
+
+
+@register.simple_tag
+def breadcrumb_items(hierarchy: list, is_tna: bool, current_item: Record) -> list:
```

Comment:
It would be nice to see some test cases for this method.

Reply:
```suggestion
def breadcrumb_items(hierarchy: list, is_tna: bool, record: Record) -> list:
```

I dont think current_item is passed see `{% breadcrumb_items record.hierarchy record.is_tna record as breadcrumbs %}` it should be named as just record instead of current_item.